components/Heading/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const sizes = {
+  xl: 'govuk-heading-xl',
+  l: 'govuk-heading-l',
+  m: 'govuk-heading-m',
+  s: 'govuk-heading-s'
+};
+
+const defaults = {
+  h1: 'xl',
+  h2: 'l',
+  h3: 'm',
+  h4: 's'
+};
```

Comment:
would it be worth extracting these into a constants file?

Reply:
At the moment these are only used within this component, if they are used more widely then agree we should look at either exporting them or moving them out - but for the current use I think this is fine.